### DISCUSSION TITLE
My 240*320 (ILI9341) not works.

### DISCUSSION THREAD

**Discussion by zealcrew** [Bodmer/TFT_eSPI#1796] [Compatible displays and setup files]
Hello, Dear Bodmer and other experts!
I got this LCD display (pictures attached).  I can't understand how to connect it to work with TFT_SPI. The display has an LCD_WR pin. I can't understand: if it works with SPI, then what is the LCD_WR pin for? I tried many variations of the config settings before contacting you. I really like your library and I use it a lot with ESP8266 (WEMOS D1 mini). I ask for help in launching this display with your library.

Display:
2,4" SPI TFT Display 240*320 (ILI9341) Ver. 1

Pinout:
5V
3.3V
LCD_MOSI
LCD_SCK
LCD_SS (LCD_CS)
LCD_WR
LCD_DC
LCD_RUS
LCD_LED
GND
![3153_image](https://user-images.githubusercontent.com/73593477/165127585-80ca5562-fd89-415c-9336-59f94bba795f.jpeg)
![837](https://user-images.githubusercontent.com/73593477/165127640-e30bccd9-a6c8-4985-940e-c6abb40cf318.png)

**Comment by Bodmer** [comment]
That customised display interface is not supported.
